### PR TITLE
Add Button

### PR DESCRIPTION
**Pull request by inductionotg**

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+.button:hover{
+    background-color: #008168;
+}
+.button:active{
+    background-color:#006e58;
+}
+.button__text,.button__icon{
+    display: inline-flex;
+    align-items: center;
+    padding: 0 24px;
```
Comment (ravikumar1002) [inline components/button/button.css]: use rem instead of px

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+.btn-outline-error:hover{
+    background-color: var(--error-color);
+    color: var(--light-color);
+}
+/* button class for the butto with icons*/
+.button{
+    display: inline-flex;
+    height:50px;
+    background-color: #009578;
```
Comment (ravikumar1002) [inline components/button/button.css:148]: use color variable

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+}
+
+.btn-outline-error:hover{
+    background-color: var(--error-color);
+    color: var(--light-color);
+}
+/* button class for the butto with icons*/
+.button{
+    display: inline-flex;
+    height:50px;
```
Comment (ravikumar1002) [inline components/button/button.css:147]: use rem instead of px

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    flex-wrap: wrap;
+    margin: 1rem;
+}
+
+.btn{
+    background-color: inherit;
+    border-radius: 0.625rem;
+    border: none;
+    height: 2rem;
+    width: 8rem;
```
Comment (ravikumar1002) [inline components/button/button.css:13]: One suggestion,  don't fix size of button

File: components/button/button.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+/* button class for the butto with icons*/
+.button{
+    display: inline-flex;
+    height:50px;
+    background-color: #009578;
+    border: none;
+    outline: none;
+    border-radius:5px ;
+    overflow: hidden;
+    font-size: 16px;
```
Comment (ravikumar1002) [inline components/button/button.css:153]: Try to using one unit

**Review by ravikumar1002 — APPROVED** [review]

File: components/button/button.css (hunk 5 above)
Comment (inductionotg) [reply components/button/button.css:153]: ok..i will take care of your feedback.. thank you for the feedback

**Pull request merged**